### PR TITLE
Port MineDigger to fn3

### PR DESCRIPTION
**Pull request by abarth**
@Hixie

### CODE REVIEW

File: examples/mine_digger/lib/main.dart [hunk 1]
```diff
@@ -170,18 +175,18 @@ class MineDiggerApp extends App {
...
-        child: new Text(toolbarCaption, style: Theme.of(this).text.title)
+        child: new Text(toolbarCaption, style: Theme.of(context).text.title)
       )
     );
   }
 
-  Widget build() {
+  Widget build(BuildContext context) {
     // We build the board before we build the toolbar because we compute the win state during build step.
     Widget board = buildBoard();
```
Comment (Hixie) [inline examples/mine_digger/lib/main.dart]: convert this to using the new App component.

File: sky/packages/updater/.gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.idea
```
Comment (Hixie) [inline sky/packages/updater/.gitignore]: did you mean to check this in?

**Comment by Hixie** [comment]
LGTM

File: sky/packages/updater/.gitignore (hunk 2 above)
Comment (abarth) [inline sky/packages/updater/.gitignore]: Yeah, I put it in a separate commit.
